### ISSUE TITLE
Auto-capitalisation

### ISSUE THREAD

**Issue by mark-wade** [ckeditor/ckeditor4#4115]
## Type of report

Bug

(Copy of https://dev.ckeditor.com/ticket/12142)

## Provide detailed reproduction steps (if any)

1. Be using a Mac or iOS device with automatic capitalisation enabled (I'm not sure if other operating systems have a similar option)

<img width="780" alt="macOS" src="https://user-images.githubusercontent.com/1433869/85083582-82fe9080-b215-11ea-9689-9b662fb03eae.png">

![IMG_33770E2FA1F7-1](https://user-images.githubusercontent.com/1433869/85083749-015b3280-b216-11ea-9c14-c57df93b7524.jpeg)

2. Go to https://ckeditor.com/ckeditor-4/demo/
3. At the start of the editor, type any letter.
4. Start a new sentence and type any letter.
4. Start a new paragraph and type any letter.

### Expected result

Letters at the start of editor, new paragraphs and new sentences should be capitalised.

### Actual result

They're not.

## Other details

Reproduced on

* Browser: Safari 13.1.1 (15609.2.9.1.2)
* OS: macOS 10.15.5 (19F101)
* CKEditor version: Whatever is on that demo page (at time of writing latest CKEditor 4 version is 4.14.1)
* Installed CKEditor plugins: Whatever is on that demo page

**Comment by jswiderski** [comment]
From what I have checked this Mac setting works for e.g. Notes application, however, it doesn't work in contenteditable elements at all.  I have tested it inide `<div contenteditable='true'></div>` (so no editor code has be used in the test) element and it didn’t capitalize anything.

I have also tested plain textarea and there was no capitalization as well.

This is not something we can fix as this Mac feature doesn't seem to work on web pages.
